clients/ts/FunctionalTests/ts/TestLogger.ts

```diff
@@ -1,4 +1,7 @@
-import { ConsoleLogger, ILogger, LogLevel } from "@aspnet/signalr";
+import { ILogger, LogLevel } from "@aspnet/signalr";
+
+// Pull some useful internal utilities from the signalr library.
```

Comment:
This is a sneaky thing tests can do. Since JavaScript imports work on the file level, we can actually just grab internals straight out of the file in which they are defined.

Reply:
Put that as a comment in the ts file